doc/nom-guide/src/chapter_1.md

```diff
@@ -0,0 +1,82 @@
+# Chapter 1: The Nom Way
+
+First of all, we need to understand the way that regexes and nom think about
```

Comment:
I am not sure starting by comparing with regexes helps in understanding. We should be able to explain parser combinators independently (ie, parsers, so takes text or bytes as input and returns structured data and combinators, taking smaller parsers and making larger ones), then extend to a comparison with other approaches likes regular expressions (powerful, but limited to regular languages).
Otherwise we are already assuming that the reader knows enough about regular expressions, which might not be the case.

Reply:
Agreed, and modified :)